protocols/cc_protocol.py

```diff
@@ -11,13 +11,13 @@
 class CCProtocol:
 
     fig_2_base_cmd_fmt = "mm-delay {delay} \
-            mm-link --uplink-log={uplink_log} \
+            mm-link --once --uplink-log={uplink_log} \
             {queue_args} \
             {uplink} {downlink} \
-            {mahimahi_command}"
+            -- bash -c '{mahimahi_command}'"
 
     fig_2_results_cmd_fmt = "mm-throughput-graph 500 {log_file} > \
-            /dev/null 2> {results_file}"
+            graph.svg 2> {results_file}"
```

Comment:
This could maybe a command line argument, but probably we don't always want to generate the graph?

Reply:
I can add a command line argument